docs/content/rules/sort-imports.mdx

```diff
@@ -201,6 +201,12 @@ Specifies a maximum line length for sorting imports. When the line length exceed
 
 This option is only available when the type is set to `'line-length'`.
 
+### tsConfigRootDir
+
+<sub>default: `undefined`</sub>
```

Comment:
what about defaulting to something else, like:
- `languageOptions.parserOptions.tsconfigRootDir` (<=> `undefined` if not defined)
- `import.meta.dirname` (with the possibility to override with `null` if someone isn't interested in such an import sorting)

see https://typescript-eslint.io/getting-started/typed-linting/